### PR TITLE
Upgrade Nexus OCI image to 3.70.2

### PR DESCRIPTION
**Pull request by EugenMayer**

### CODE REVIEW

**Review by stevehipwell — APPROVED** [review]
Thanks for the PR @EugenMayer, but you're missing a couple of required changes. You can either copy the last PR for this chart or I can re do the PR?

**Comment by EugenMayer** [comment]
I check the old one and adjust it, thanks for the hint

**Comment by EugenMayer** [comment]
@stevehipwell looking at https://github.com/stevehipwell/helm-charts/pull/1008/files#diff-dcea0eacf433f10dc1eb3446d638828de4784506601e5573d72df989b508606e i cannot see what is missing - could you hint me?

**Comment by stevehipwell** [comment]
@EugenMayer you're missing the release link in the changelog and chart annotations (see https://github.com/stevehipwell/helm-charts/pull/969).

**Comment by EugenMayer** [comment]
@stevehipwell thanks, will follow up

they did not release it on gh, just on docker hub .. why so ever. I do not understand how they managed to do so, but the .2 release is a important patched one for the migration path.

**Comment by EugenMayer** [comment]
> @EugenMayer you're missing the release link in the changelog and chart annotations (see #969).

Not sure whay all this is under unreleased and what this link actually does, but nevertheless, added what you asked for (hoping so)

**Review by stevehipwell — CHANGES_REQUESTED** [review]
You're still missing the changes to the chart annotations. If you add that and bump the date to the 6th I can release it tomorrow.

**Comment by EugenMayer** [comment]
Fixed what you asked for

File: charts/nexus3/CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,14 @@
 
 ## [UNRELEASED]
 
+## [v4.45.1] - 2024-09-06
+
+### Addded
+
+### Changed
+
+- Updated the _Nexus3_ OCI image to [v3.70.2](https://hub.docker.com/layers/sonatype/nexus3/3.70.2-java11-alpine/images/sha256-ae90e722a1d07b8810482e78be352686450cd468bd87529c1ca2e70468f0d36f?context=explore).
```
Comment (stevehipwell) [inline charts/nexus3/CHANGELOG.md:23]:
```suggestion
- Updated the _Nexus3_ OCI image to `v3.70.2`.
```
The above is incorrect, the chart uses the `java11-ubi` image by default. Also the pattern here is to link to the source repo, so if the release is missing I'd prefer to remove the link.

File: charts/nexus3/Chart.yaml [hunk 2]
```diff
@@ -23,9 +23,5 @@ maintainers:
 annotations:
   artifacthub.io/alternativeName: nexus
   artifacthub.io/changes: |
-    - kind: added
-      description: "Added `service.additionalPorts.hosts` (list) for multi-hosts support."
     - kind: changed
-      description: "Updated the _Nexus3_ OCI image to [v3.70.1](https://github.com/sonatype/nexus-public/releases/tag/release-3.70.1-02)."
-    - kind: deprecated
-      description: "Deprecated `service.additionalPorts.host` in favour of `service.additionalPorts.hosts`."
+      description: "Updated the _Nexus3_ OCI image to [v3.70.2](https://hub.docker.com/layers/sonatype/nexus3/3.70.2-java17-alpine/images/sha256-2673b4f2ad59838d87ea066b6a275d4520b0aa1c0271f24954276eaa6686adf0?context=explore)."
```
Comment (stevehipwell) [inline charts/nexus3/Chart.yaml:27]:
```suggestion
      description: "Updated the _Nexus3_ OCI image to `v3.70.2`."
```
See above (not that it matters here as all markdown is stripped)

**Review by stevehipwell — CHANGES_REQUESTED** [review]

File: charts/nexus3/Chart.yaml (hunk 2 above)
Comment (EugenMayer) [reply charts/nexus3/Chart.yaml]: fixed

File: charts/nexus3/CHANGELOG.md (hunk 1 above)
Comment (EugenMayer) [reply charts/nexus3/CHANGELOG.md]: fixed

**Review by stevehipwell — APPROVED** [review]
LGTM

**Pull request merged**